app/delivery/send_to_providers.py

```diff
@@ -148,6 +148,10 @@ def update_notification_to_sending(notification, provider):
     dao_update_notification(notification)
 
 
+provider_cache = TTLCache(maxsize=8, ttl=2)
```

Comment:
TTL 2 seconds? Yet you say in your comment `I think this shows that you really start to see the benefit with the 10s cache, where >50% of queries are skipping the database.`. Does that not encourage a longer TTL?

Reply:
Changed to 10 seconds in 55afc9a